### DISCUSSION TITLE
Italic Font

### DISCUSSION THREAD

**Discussion by decoomanj** [psb1558/Junicode-font#52] [Q&A]
Hello, I'm currently using your font in a LaTeX document, but I see that the italics font is in the legacy folder, and is missing in the fonts folder? In LaTeX I need to specify the italic-font explicitly.  Or is there another way to use italics?

Keep up the good work!
Jan

**Comment by psb1558** [comment]
There is no italic yet for JuniusX because it is nowhere near ready for production use. If I posted JuniusX-Italic, which is still incomplete, the free font sites would quickly pick it up, and they don't usually upgrade when new versions are released, so a broken version would be in circulation essentially forever. If you have Glyphs (the font editor), though, or if you know someone who does, you can generate the incomplete italic for yourself.

**Reply by decoomanj** [reply]
Unfortunately I can't run Glyphs on Linux. I'll have to fall back to the legacy font for now. Thanks and looking forward to the final release!

**Reply by psb1558** [reply]
I should have mentioned that it may compile with fontmake, which certainly does run on Linux. Unfortunately, I haven't yet tried this with the italic, though I know it works with the roman. But for now, legacy Junicode has the fuller character set in the italic.

**Comment by sakkamade** [comment]
> If I posted JuniusX-Italic, which is still incomplete, the free font sites would quickly pick it up

I was thinking, being that its name is not yet decided (https://github.com/psb1558/Junicode-font/issues/66), why not start releasing the italic together with regular while the font is under development with this name, and then rename it when it is ready-to-use?

Such web-sites would also be compelled to  quickly pick up that one, too, since it will be under another name, right? 

Also, (from here https://github.com/psb1558/Junicode-font/issues/65#issuecomment-915418807) because such web-sites are most likely already having the former Junicode, they will not update it either way. (If it will retain the name of Junicode.)

**Reply by psb1558** [reply]
I've decided to retain the name Junicode (I don't like it much, but it's well known) and bump the major version to 2. Perhaps I should use a name like JunicodeBeta or even JunicodeTwoBeta while it's in development, to make it utterly clear what the status is, and make it more-or-less safe to provide the italic (which is far enough along, anyway, that I can use it myself).

And incidentally allow it to be installed alongside Junicode 1.

**Comment by sakkamade** [comment]
> I've decided to retain the name Junicode (I don't like it much, but it's well known) and bump the major version to 2.

This is a very right thing to do. The same approach, for example, has been employed throughout a years by [Gentium](https://software.sil.org/gentium/history/), although it has also a "Plus"-version now.

> JunicodeBeta or even JunicodeTwoBeta

Latter should be nice, I think, however——

> And incidentally allow it to be installed alongside Junicode 1.

However, I believe that this is not so good.

Junicode One and Junicode Two, would seem to be a better choice, as the versioning should not go anywhere, i.e. Junicode Two of version e.g. "0.004", or "1.000 alpha", in my opinion, should more plainly denote its being as yet under development.

I am of opinion that versioning (the "tags") is very important part for either developer and (https://github.com/psb1558/Junicode-font/issues/65) packagers/users.

**Reply by sakkamade** [reply]
> or "1.000 alpha",

If you plan to go with this one, you can also play with that word. 
For example, 1.010 alpha, 1.020 alpha, then can even be the 1.000 beta, gamma, delta, etc.
